src/main/java/net/sf/jabref/logic/importer/ImportException.java

```diff
@@ -0,0 +1,17 @@
+package net.sf.jabref.logic.importer;
+
+public class ImportException extends Exception {
```

Comment:
Why do we need a new Exception here? I mean, what is the benefit of knowing that this is specific to the importer? My main point is that we do not really have a strategy as to when we add a new exception and when to use a specific exception.

How about just reusing the existing JabRefException? Does the job as well, as far as I see.